src/storage-types/src/dyncfgs.rs

```diff
@@ -256,6 +256,13 @@ pub const STORAGE_SERVER_MAINTENANCE_INTERVAL: Config<Duration> = Config::new(
     "The interval at which the storage server performs maintenance tasks. Zero enables maintenance on every iteration.",
 );
 
+/// The interval at which the storage server performs maintenance tasks.
```

Comment:
"Whether sinks iteratively search through the progress topic, with exponentially larger lookback."

Reply:
🤦 - thanks.